gcp/rakefiles/secrets.rb

```diff
@@ -96,13 +101,19 @@ def self.set_secrets(collected_secrets, exekube_cmd)
           populated_secrets = {}
           secrets.each do |secret_name|
             if ENV[secret_name.upcase].to_s.empty?
-              secret_value = SecureRandom.hex
+              if secret_name.match(/^(key)_/)
```

Comment:
Do you need the parentheses?

Reply:
Nope, I copy-pasted it from somewhere else.